crates/telio-dns/src/dns.rs

```diff
@@ -28,7 +28,7 @@ pub trait DnsResolver {
     /// Configure list of forward DNS servers for zone '.'.
     async fn forward(&self, to: &[IpAddr]) -> Result<(), String>;
     /// Get public key of this DNS server.
-    fn public_key(&self) -> PublicKey;
+    fn public_key(&self) -> [u8; 32];
```

Comment:
I would prefer that you would leve telio_crypto::PublicKey in public interfaces, it helps people understand what data they are working with